src/lib/webpack-config.js

```diff
@@ -208,6 +216,37 @@ export default env => {
 			}
 		}),
 
+		// TypeScript
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
```

Comment:
In theory that should work, but somehow babel-loader is skipped if I don't explicitly add the babel config here. Perhaps @s-panferov or @TheLarkInn may help here.